Dockerfile

```diff
@@ -8,17 +8,9 @@ FROM docker:$DOCKERD_VERSION AS dockerd-release
 # xx is a helper for cross-compilation
 FROM --platform=$BUILDPLATFORM tonistiigi/xx@sha256:21a61be4744f6531cb5f33b0e6f40ede41fa3a1b8c82d5946178f80cc84bfc04 AS xx
 
-FROM --platform=$BUILDPLATFORM golang:1.16-alpine AS golatest
+FROM --platform=$BUILDPLATFORM golang:1.17-alpine AS golatest
```

Comment:
Should we have a test-matrices to test `current` and `n-1` versions of go?

also see https://github.com/docker/cli/pull/3258#issuecomment-900332386

Reply:
> Should we have a test-matrices to test `current` and `n-1` versions of go?

Good idea I will add a dedicated matrix in a follow-up.

> also see [docker/cli#3258 (comment)](https://github.com/docker/cli/pull/3258#issuecomment-900332386)

Ah yes buildx is rebuilt through the packaging pipeline. Forgot about that.

Reply:
I'm not sure we need a matrix. This is a cli tool. Good pattern to leave them configurable though.